config_scripts/parse_args.py

```diff
@@ -15,7 +15,9 @@ def parse_args():
     for args_lst in COMMAND_LINE_PARAMS:
         argument, arg_type, arg_help = args_lst[0], args_lst[1], args_lst[2]
         parser.add_argument(f'--{argument}', type=arg_type, help=arg_help)
+        parser.add_argument(f'-{argument}', type=arg_type, help=arg_help)
 
+    parser.add_argument('-optimize', action='store_true', help='Enable optimizations')
```

Comment:
Generalize this with another loop. Add full support for SB3 command line parameters, specifically RLZoo3.